spec/factories/post.rb

```diff
@@ -0,0 +1,21 @@
+require 'faker'
+
+FactoryBot.define do
+  factory :post do
+    title { Faker::Lorem.sentence(word_count: 4) }
+    body { Faker::Lorem.sentences(number: 5) }
+
+    trait :with_no_body do
+      body { '' }
```

Comment:
instead of `FactoryBot.create(:post, :with_no_body)` you could simply have `FactoryBot.create(:post, body: '')`, so not sure why this traits are needed